<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>grm_lsi</class>
 <widget class="QWidget" name="grm_lsi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GRM - LSI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/grass/qtui/icons/grass/grass_location.svg</normaloff>:/grass/qtui/icons/grass/grass_location.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources.qrc">:/grass/qtui/icons/grass/grass_location.svg</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Compute metrics for LSI bedforms</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>507</width>
        <height>812</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>70</height>
          </size>
         </property>
         <property name="title">
          <string>Input</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <widget class="QLabel" name="label_32">
            <property name="text">
             <string>Name of input elevation raster map:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="elevation"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>320</height>
          </size>
         </property>
         <property name="title">
          <string>SWC</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Inner search radius</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="swc_dist">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Flatness distance, zero for none:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="iter_thin">
            <property name="text">
             <string>400</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Thinning iterations</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>v.generalize threshold</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="swc_skip">
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>Min area threshold</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="generalize_method">
            <item>
             <property name="text">
              <string>douglas</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>douglas_reduction</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>lang</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>reduction</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>reumann</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>boyle</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>sliding_averaging</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>chaiken</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>hermite</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>snakes</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>network</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>displacement</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="swc_search">
            <property name="value">
             <number>9</number>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="swc_area_lesser">
            <property name="text">
             <string>70</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="swc_flat">
            <property name="text">
             <string>2.0</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>v.generalize method</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="generalize_threshold">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>Outer search radius:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>Flatness threshold (degrees):</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>v.clean rmdangle threshold</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="vclean_rmdangle_threshold">
            <property name="text">
             <string>5,10,20,30</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>220</height>
          </size>
         </property>
         <property name="title">
          <string>SW</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="1">
           <widget class="QSpinBox" name="sw_search">
            <property name="value">
             <number>30</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_33">
            <property name="text">
             <string>Flatness distance, zero for none:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="sw_flat">
            <property name="text">
             <string>3.8</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_34">
            <property name="text">
             <string>Flatness threshold (degrees):</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="sw_skip">
            <property name="value">
             <number>7</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_35">
            <property name="text">
             <string>Outer search radius:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="sw_area_lesser">
            <property name="text">
             <string>1000</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_36">
            <property name="text">
             <string>Min area threshold</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_37">
            <property name="text">
             <string>Inner search radius</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="sw_dist">
            <property name="text">
             <string>15</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_38">
            <property name="text">
             <string>v.clean rmarea threshold</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="vclean_rmarea_threshold">
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>180</height>
          </size>
         </property>
         <property name="title">
          <string>SW Metrics</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="2" column="1">
           <widget class="QLineEdit" name="point_dmax">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_28">
            <property name="text">
             <string>Side-split buffer distance</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>Transect split length</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="transect_split_length">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_30">
            <property name="text">
             <string>Point max distance</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="buffer_distance">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_31">
            <property name="text">
             <string>Transects side distance</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="transect_side_distances">
            <property name="text">
             <string>70,70</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="run">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="reload_layers">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/fa_solid/qtui/icons/fa_solid/arrows-rotate.svg</normaloff>:/fa_solid/qtui/icons/fa_solid/arrows-rotate.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exit">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="add_output">
     <property name="text">
      <string>Add Geomorphic forms into layer tree</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
